### DISCUSSION TITLE
After update to AIO 6.1.0, no login possible - Update required

### DISCUSSION THREAD

**Discussion by Moonbase59** [nextcloud/all-in-one#2715] [Questions] [answered]
### Steps to reproduce
1. Update AIO (beta) to 6.1.0
2. Use "Open your Nextcloud" button
3. No login possible.

### Expected behavior 
Normal login

### Actual behavior 
Website shows only
```
Update wird benötigt
Da automatische Updates in der config.php deaktivert sind, nutze bitte den Updater über die Kommandozeile.
```
No login possible.

### Host OS 
Ubuntu Server 22.04.2 LTS

#### Nextcloud AIO version 
6.1.0 (after update)

#### Current channel 
beta

#### Other valuable info 

I have no idea what to do. All the "occ" things in the docs don’t work, because AIO is installed via Docker. There were no issues with previous updates.

I _had_ access very shortly (because being logged in) and tried to do the requested updates for "Calendar" and "Contacts", which both failed with an error message. I then logged out and saw above message when trying to log back in.

I already tried stopping/starting the containers using the local AIO site, and even upgraded and restarted the whole VM. No luck.

The AIO site shows that actually everything should be up-to-date:

![AIO – Mozilla Firefox_001](https://github.com/nextcloud/all-in-one/assets/3706922/d44fe62e-8fcc-46a4-8c0d-2c7ffb5fe18d)

**Comment by DenuxPlays** [comment]
Same error
Also happend after trying to update calendar and contacts app.

**Comment by DenuxPlays** [comment]
Downgrading to all-in-one stable doesn't solve the issue...

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1787
